### DISCUSSION TITLE
how to generate ssl_server_cert_dn ? and where do we need to pass?

### DISCUSSION THREAD

**Discussion by prashanttest-elastic** [oracle/python-oracledb#137] [Q&A]
I want pass `ssl_server_cert_dn` with my custom SSL certificate. please suggest me how to generate ssl_server_cert_dn and where we pass in oracledb (thin model).

**Comment by cjbj** [comment]
The generation question would be best asked to some security people.   

You can specify ssl_server_cert_dn when creating a [connection](https://python-oracledb.readthedocs.io/en/latest/api_manual/module.html#oracledb.connect) or [pool](https://python-oracledb.readthedocs.io/en/latest/api_manual/module.html#oracledb.create_pool), or you can pass it in a connect descriptor, e.g. in a tnsnames.ora file. 

Let us know if you get it working.
